bin/miden-cli/src/commands/export.rs

```diff
@@ -116,9 +116,10 @@ fn export_note<N: NodeRpcClient, R: FeltRng, S: Store, A: TransactionAuthenticat
     let note_file = match export_type {
         ExportType::Id => NoteFile::NoteId(output_note.id()),
         ExportType::Full => match output_note.inclusion_proof() {
-            Some(inclusion_proof) => {
-                NoteFile::NoteWithProof(output_note.clone().try_into()?, inclusion_proof.clone())
-            },
+            Some(inclusion_proof) => NoteFile::NoteWithProof(
+                output_note.clone().try_into().map_err(|err: NoteRecordError| err.to_string())?,
```

Comment:
Can we use the `?` operator directly here?
```suggestion
                output_note.clone().try_into()?,
```

Reply:
Added a conversion to `String` so we could automatically cast the error